qemu/tests/cfg/remote_image_multiwrite.cfg

```diff
@@ -0,0 +1,31 @@
+# Network storage backends:
+#   iscsi_direct
+#   ceph
+#   gluster_direct
+# The following testing scenario is covered:
+#   Format a data disk with ext4 in VM.
```

Comment:
with ext4 and xfs

Reply:
I mean add xfs into line 6. only ext4 in comment.

Reply:
Yes, I got your point. I have already updated the code and pushed the branch, but weird, I didn't find the code update here.....
I'll push it again, and see if it works.

Reply:
it works now, please have a check, thanks.